core/amber/src/main/scala/edu/uci/ics/amber/engine/architecture/controller/Controller.scala

```diff
@@ -99,4 +103,13 @@ class Controller(
   }
 
   override def handleBackpressure(isBackpressured: Boolean): Unit = {}
+
+  override val supervisorStrategy: SupervisorStrategy =
```

Comment:
if you take this solution from an online post, can you leave the source url in the comment?

Reply:
done